phing/tasks/local-sync.xml

```diff
@@ -15,15 +15,21 @@
     </phingcall>
   </target>
 
-  <target name="local:sync" description="Synchronize local environment from remote (remote --> local)."
+  <!-- Sync each local multisite from remote aliases. -->
+  <target name="local:sync" description="Synchronize each local environment from remote (remote --> local).">
+    <yamlKeys file="${blt.config-files.project}" variable="multisite" outputProperty="multisite.names"/>
+    <foreach list="${multisite.names}" param="multisite.name" target="local:sync:site"/>
```

Comment:
Rather than apply this to every multisite every time that this target is run, I'd prefer to run one command per multisite. Why not just allow the user the pass the multisite name as an argument?

Better yet, we can avoid a lot of the configuration nesting if we use multiple config files, one per multi site. Each multisite can have a unique file that defines drush.aliases, project.uri, etc. You can pass a reference to the config file using `-propertyfile <file> -propertyfileoverride`. If that's too much to type, we can wrap *that* with something higher level.